README.md

```diff
@@ -1,4 +1,32 @@
-# Northcoders News API
+Project Summary - # Northcoders News API
+A project building the backend module of the Northcoders Bootcamp.
+
+The project involved building an API for the purpose of accessing application data programmatically. The intention here was to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
+
+The database is PostgreSQL (PSQL), and interaction with it is carried out using node-postgres.
+
+The project involved building of multiple endpoints that made use of Create, Read, Update, and Delete (CRUD)operations. Endpoints are listed below:
+
+app.get("/api/topics", getAllTopics);
+app.get("/api/articles/:article_id", getArticleById);
+app.get("/api/articles", getAllArticles);
+app.get("/api/articles/:article_id/comments", getCommentsByArticleId);
+app.post("/api/articles/:article_id/comments", postCommentsByArticleId);
+app.patch("/api/articles/:article_id", patchArticleByArticleId);
+app.delete("/api/comments/:comment_id", deleteCommentByCommentId);
+
+The project reinforced GitHub learning and utilised git branching and pull requests for code reviews. Merging of pull requests into the main branch was also performed.
+
+Web Links
+Link to the hosted version on Render https://issh-nc-news.onrender.com/api
+PostgreSQL database is hosted on SupabaseL https://zxtiguqhfypnzklkkohr.supabase.co
```

Comment:
I know this isn't some big scale security project but typically I wouldn't put your production database URL in a readme.